srcpkgs/muse/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'muse'
+pkgname=muse
+version=3.1.1
+revision=1
+#archs="i686 x86_64"
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="alsa-lib-devel dssi-devel fluidsynth-devel gtkmm2-devel jack-devel ladish-devel
+ ladspa-sdk liblo-devel liblrdf-devel libsamplerate-devel libsndfile-devel lilv-devel
+ python3-devel qt5-devel qt5-svg-devel qt5-tools-devel rubberband-devel sord-devel"
+depends="desktop-file-utils python3-six python3-Pyro4"
+short_desc="MIDI/Audio sequencer with recording and editing capabilities"
+maintainer="Evgeny Ermakov <evgeny.v.ermakov@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://muse-sequencer.github.io/"
+_name="${pkgname}-${version}"
+distfiles="https://github.com/muse-sequencer/${pkgname}/releases/download/${_name//[-.]/_}/${_name}.tar.gz"
+checksum=d40126543d549af30090d1f98422588d74ad1e336ad43e911bd4b02eff217842
+nocross="https://travis-ci.org/github/void-linux/void-packages/jobs/734779029"
+
+post_install() {
+	for file in ${DESTDIR}/usr/share/muse-3.1/utils/*; do
+		vsed -i -e 's,/usr/bin/python,/usr/bin/python3,' $file
+	done
```

Comment:
There is already a hook that does this... if it can't autodetect the version, set `python_version=3`

Reply:
Thanks, it worked!